data/events.yaml

```diff
@@ -0,0 +1,610 @@
+x-events:
+  - name: generic_cluster
+    message: "{message}"
+    event_type: cluster
+    severity: "info"
+    properties:
+      cluster_id: UUID
+      message: string
+      severity: string
+
+  - name: generic_host
+    message: "{message}"
+    event_type: host
+    severity: "info"
+    properties:
+      cluster_id: UUID
+      host_id: UUID
+      message: string
+      severity: string
+
+  - name: cancel_install_failed_start
+    message: "Failed to cancel installation: error starting DB transaction"
+    event_type: cluster
+    severity: "error"
+    properties:
+        cluster_id:  UUID
```

Comment:
@arielireni let's maintain a common indentation of 2 spaces offset

Reply:
solved